shared/more/style-sheet.render.native.js

```diff
@@ -178,10 +176,40 @@ const Inputs = () => (
   <Text type='Header'>TODO</Text>
 )
 
+const Checkboxes = ({check, flip}) => {
+  return (
+    <Box>
+      {false && <Row><Switch onTintColor={globalColors.blue} value={check[0]} onValueChange={() => flip(0)}/></Row>}
```

Comment:
this false stuff is just so we can compare it vs the native components. might as well leave this